packages/wallet-ui/src/components/ui/organism/Header/SendSummaryModal/SendSummaryModal.view.tsx

```diff
@@ -83,6 +86,9 @@ export const SendSummaryModalView = ({
           callData,
           wallet.accounts[0] as unknown as string,
           chainId,
+          selectedFeeToken === 'ETH'
+            ? constants.TRANSACTION_VERSION.V2
+            : constants.TRANSACTION_VERSION.V3,
```

Comment:
do you think it is better not using version in estimate fee?

as from starknet team advice , dapp should not require to tell the tx is v2 or v3, it should control by wallet

but i think they assume we follow the same exp as AgentX

Reply:
as discussed in the stand up, although dapp do not require we can do both. Since this part will only be for the wallet-ui it's fine. When doing txv3 support for executeTransaction we will have to implement a similar flow than AgentX